### ISSUE TITLE
NuGet Package Manager fails to install the cef.redist.x86 package in Visual Studio Community 2015

### ISSUE THREAD

**Issue by Snorvarg** [cefsharp/CefSharp#1511]
Hi, 
I tried to install the CefSharp.WinForms into my plain Windows Form project, but it fails somehow. After trying to install the depending packages one by one, I found that the cef.redist.x86 package fails to install with the following output from the Package Manager:

```
Attempting to gather dependencies information for package 'cef.redist.x86.3.2454.1344' with respect to project 'ChromeApp', targeting '.NETFramework,Version=v4.6'
Attempting to resolve dependencies for package 'cef.redist.x86.3.2454.1344' with DependencyBehavior 'Lowest'
Resolving actions to install package 'cef.redist.x86.3.2454.1344'
Resolved actions to install package 'cef.redist.x86.3.2454.1344'
Install failed. Rolling back...
Package 'cef.redist.x86 3.2454.1344' does not exist in project 'ChromeApp'
Package 'cef.redist.x86 3.2454.1344' does not exist in folder 'E:\Use\Programming\ChromeApp\ChromeApp\packages'
System.NullReferenceException: Object reference not set to an instance of an object.
   at NuGet.Protocol.Core.v3.GlobalPackagesFolderUtility.<AddPackageAsync>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NuGet.Protocol.Core.v3.DownloadResourceV3.<GetDownloadResourceResultAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.PackageManagement.PackageDownloader.<GetDownloadResourceResultAsync>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at NuGet.PackageManagement.NuGetPackageManager.<ExecuteNuGetProjectActionsAsync>d__42.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at NuGet.PackageManagement.NuGetPackageManager.<ExecuteNuGetProjectActionsAsync>d__42.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.PackageManagement.UI.UIActionEngine.<ExecuteActionsAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NuGet.PackageManagement.UI.UIActionEngine.<PerformActionAsync>d__3.MoveNext()
========== Finished ==========
```

...what am I doing wrong? I downloaded the CefSharp.MinimalExample and could build and execute it with no problems, but I would like to be able to start a new project from scratch. 

Thank's for all the great work with CefSharp!

**Comment by Snorvarg** [comment]
I tried to change to only x64 as proposed in this article: http://www.codeproject.com/Articles/1058700/Embedding-Chrome-in-your-Csharp-App-using-CefSharp

..But the same error still happens.

**Comment by amaitland** [comment]
If you search through previous issues for Nuget, someone had a similar problem, had to upgrade to a pre release of Nuget.

**Comment by Snorvarg** [comment]
Thanks for answering. Can you link to it? When I search for nuget among closed issues I get 221 hits. :)

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/issues/1419#issuecomment-156990395

**Comment by amaitland** [comment]
@Snorvarg Were you able to resolve the problem?

**Comment by Snorvarg** [comment]
Yep. :) I downloaded and installed v3.3.0 of nuget, and then I could install cefsharp without any problems. App is now running fine, thank's a lot!

Attempting to gather dependencies information for package 'CefSharp.WinForms.45.0.0' with respect to project 'ChromeApp', targeting '.NETFramework,Version=v4.6'
Attempting to resolve dependencies for package 'CefSharp.WinForms.45.0.0' with DependencyBehavior 'Lowest'
Resolving actions to install package 'CefSharp.WinForms.45.0.0'
Resolved actions to install package 'CefSharp.WinForms.45.0.0'
Package 'cef.redist.x64.3.2454.1344' already exists in folder 'E:\Use\Programming\ChromeApp\ChromeApp_trasig\packages'
Added package 'cef.redist.x64.3.2454.1344' to 'packages.config'
Successfully installed 'cef.redist.x64 3.2454.1344' to ChromeApp
Package 'cef.redist.x86.3.2454.1344' already exists in folder 'E:\Use\Programming\ChromeApp\ChromeApp_trasig\packages'
Added package 'cef.redist.x86.3.2454.1344' to 'packages.config'
Successfully installed 'cef.redist.x86 3.2454.1344' to ChromeApp
Package 'CefSharp.Common.45.0.0' already exists in folder 'E:\Use\Programming\ChromeApp\ChromeApp_trasig\packages'
Added package 'CefSharp.Common.45.0.0' to 'packages.config'
Successfully installed 'CefSharp.Common 45.0.0' to ChromeApp
Package 'CefSharp.WinForms.45.0.0' already exists in folder 'E:\Use\Programming\ChromeApp\ChromeApp_trasig\packages'
Added package 'CefSharp.WinForms.45.0.0' to 'packages.config'
Successfully installed 'CefSharp.WinForms 45.0.0' to ChromeApp
========== Finished ==========
